front/app/containers/Admin/projects/project/participationContext/components/ParticipationMethodPicker.tsx

```diff
@@ -15,155 +15,173 @@ import messages from '../../messages';
+              name="participationmethod"
+              id={'participationmethod-survey'}
+              label={
+                <LabelHeaderDescription
+                  header={<FormattedMessage {...messages.createSurveyText} />}
+                  description={
+                    <FormattedMessage {...messages.createSurveyDescription} />
+                  }
+                />
+              }
+            />
+          )}
 
-    <FeatureFlag name="volunteering">
-      <ParticipationMethodRadio
-        onChange={handleParticipationMethodOnChange}
-        currentValue={participation_method}
-        value="volunteering"
-        name="participationmethod"
-        id={'participationmethod-volunteering'}
-        label={
-          <LabelHeaderDescription
-            header={<FormattedMessage {...messages.findVolunteers} />}
-            description={
-              <FormattedMessage {...messages.findVolunteersDescriptionText} />
+          <FeatureFlag name="volunteering">
+            <ParticipationMethodRadio
+              onChange={handleParticipationMethodOnChange}
+              currentValue={participation_method}
+              value="volunteering"
+              name="participationmethod"
+              id={'participationmethod-volunteering'}
+              label={
+                <LabelHeaderDescription
+                  header={<FormattedMessage {...messages.findVolunteers} />}
+                  description={
+                    <FormattedMessage
+                      {...messages.findVolunteersDescriptionText}
+                    />
+                  }
+                />
+              }
+            />
+          </FeatureFlag>
+          <Radio
+            onChange={handleParticipationMethodOnChange}
+            currentValue={participation_method}
+            value="information"
+            name="participationmethod"
+            id="participationmethod-information"
+            label={
+              <LabelHeaderDescription
+                header={<FormattedMessage {...messages.shareInformation} />}
+                description={
+                  <FormattedMessage {...messages.shareInformationDescription} />
+                }
+              />
             }
           />
-        }
-      />
-    </FeatureFlag>
-
-    <Radio
-      onChange={handleParticipationMethodOnChange}
-      currentValue={participation_method}
-      value="information"
-      name="participationmethod"
-      id="participationmethod-information"
-      label={
-        <LabelHeaderDescription
-          header={<FormattedMessage {...messages.shareInformation} />}
-          description={
-            <FormattedMessage {...messages.shareInformationDescription} />
-          }
-        />
-      }
-    />
-    <Error apiErrors={apiErrors && apiErrors.participation_method} />
-  </SectionField>
-);
+          <Error apiErrors={apiErrors && apiErrors.participation_method} />
+        </>
+      )}
+      {config.methodIsLocked && (
```

Comment:
Minor and not blocking. It might be better to use a ternary here. Mostly because it shows that both conditions can't happen at the same time (This and the block before)